src/sc/embedder/content/embedder.py

```diff
@@ -438,3 +439,33 @@ class EmbedderVideoJS(BrowserView):
 
     def __call__(self):
         return self.index()
+
+
+@indexer(IEmbedder)
+def searchable_text_indexer(obj):
+    """SearchableText should contain id, title, description, body text,
+    alternate text and keywords.
+    """
+    transformer = ITransformer(obj)
+    text = obj.text
+    if IRichTextValue.providedBy(text):
+        text = transformer(text, 'text/plain')
+    else:
+        text = ''
+
+    alternate_text = obj.alternate_content
+    if IRichTextValue.providedBy(alternate_text):
+        alternate_text = transformer(alternate_text, 'text/plain')
+    else:
+        alternate_text = ''
```

Comment:
if `alternate_content` is not a RichText will always be empty.. well.. alternate_content [will be always be RichText](https://github.com/simplesconsultoria/sc.embedder/blob/master/src/sc/embedder/content/embedder.py#L142-L146) then you can change it to:
```python
    alternate_text = transformer(obj.alternate_content, 'text/plain')
```

Reply:
`text` could be None.